tests/cbmc/proofs/s2n_stuffer_printf/s2n_stuffer_printf_harness.c

```diff
@@ -34,7 +45,10 @@ void s2n_stuffer_printf_harness()
     /* CBMC defines va_list as void** */
     size_t va_list_size;
     __CPROVER_assume(va_list_size % sizeof(void*) == 0);
+    __CPROVER_assume(va_list_size > 0);
     void** va_list_mem = malloc(va_list_size);
```

Comment:
Given the CBMC definition of vargs as just a list of void pointers, is this accurate? You technically can call vsnprintf with no vargs: https://godbolt.org/z/3jPb7sn7d

Reply:
Ah, yes, no NULL-termination here. Fixed.